assets/src/edit-story/components/panels/panel/shared/test/handle.js

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import { render } from '@testing-library/react';
+
+/**
+ * Internal dependencies
+ */
+import { ThemeProvider } from 'styled-components';
+import DragHandle from '../handle';
+import theme from '../../../../../theme';
+
+const mockBind = jest.fn();
+const mockUnbind = jest.fn();
+
+jest.mock('Mousetrap', () =>
```

Comment:
This whole mock implementation needs to be removed/adapted as Mousetrap is not used directly anymore.

Reply:
Done.